app/gui2/src/util/getIconName.ts

```diff
@@ -35,3 +36,9 @@ const typeNameToIconLookup: Record<string, Icon> = {
 export function typeNameToIcon(typeName: string): Icon {
   return typeNameToIconLookup[typeName] ?? 'enso_logo'
 }
+
+export function displayedIconOf(entry?: SuggestionEntry, actualType?: Typename): Icon {
+  if (entry?.iconName) return mapOldIconName(entry.iconName)
+  if (actualType) return typeNameToIcon(actualType)
```

Comment:
This loses the method name check in the node. I had deliberately left that in as otherwise, all functions returning a Text (for example) would get the `text_edit` icon.

Reply:
Fixed.